pennylane/labs/trotter/fragments/generic_fragments.py

```diff
@@ -0,0 +1,70 @@
+"""Wrapper class for generic fragment objects"""
+
+from __future__ import annotations
+
+from typing import Any, Callable, Dict, List, Sequence
```

Comment:
Please remove if not necessary. Please also check the rest of PennyLane for compatibility.

Reply:
Do we have this in the rest of PL (not labs)? If not, please double check if it brings any value or creates incompatibility.

Reply:
Shouldn't bring any incompatibility, it is used in many other places in PL.